frontend/src/components/pages/Scheduling/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface SchedulingFormProps {
+  category: string;
+  quantity: number;
+  size: string;
+  pickup_location: string;
```

Comment:
similar to other types (i.e. `AuthTypes.ts`) on the frontend, let's keep it as `camelCase` ?